flight/states/mast.py

```diff
@@ -0,0 +1,26 @@
+"""After the laps, goes to the mast"""
+import logging
+import asyncio
+import math
+import mavsdk as sdk
+
+from flight import config
+
+from flight.utils.movement_controller import MovementController
+from .land import Land
+
+class Mast:
+    """ Sends the drone from the first pylon to the mast """
+    async def run(self, drone):
+        """Sends the drone from the first pylon to the mast"""
+        mover: MovementController = MovementController()
+        # Go to the mast
+        logging.info("Moving to mast")
+        await mover.move_to(drone, config.mast)
```

Comment:
Note about move_to function it fly's to an offset point to the right and not to the intended point. Not your fault but something to consider. May want to look into setting offset based off of a boolean.